services/gateway/sources/version3/mappings/transaction.js

```diff
@@ -30,6 +30,7 @@ module.exports = {
 		height: '=,number',
 		timestamp: '=,number',
 	},
+	meta: '=',
 	confirmations: '=',
```

Comment:
Please make the mapping for `meta` explicit.
```suggestion
	meta: '=',
	confirmations: '=,number',
```